src/applications/drydock/storage/DrydockRepositoryOperation.php

```diff
@@ -0,0 +1,305 @@
+<?php
+
+/**
+ * Represents a request to perform a repository operation like a merge or
+ * cherry-pick.
+ */
+final class DrydockRepositoryOperation extends DrydockDAO
+  implements
+    PhabricatorPolicyInterface {
+
+  const STATE_WAIT = 'wait';
+  const STATE_WORK = 'work';
+  const STATE_DONE = 'done';
+  const STATE_FAIL = 'fail';
+
+  protected $authorPHID;
+  protected $objectPHID;
+  protected $repositoryPHID;
+  protected $repositoryTarget;
+  protected $operationType;
+  protected $operationState;
+  protected $properties = array();
+  protected $isDismissed;
+
+  private $repository = self::ATTACHABLE;
+  private $object = self::ATTACHABLE;
+  private $implementation = self::ATTACHABLE;
+  private $workingCopyLease = self::ATTACHABLE;
+
+  // TM CHANGES fetching landing gauge
+  private $landingMetric;
+
+  function __construct() {
+      $registry = PhabricatorPrometheusApplication::getRegistry();
+      $this->landingMetric = $registry->getOrRegisterGauge(
+      self::METRIC_NAMESPACE,
+      'landing_operations_total',
+      'The number of ongoing landing operations',
+      ['repo']
+      );
+  }
+  // TM CHANGES END
```

Comment:
This is defined twice here and where we decrement it in the worker ... It would be safer if we could only define it once and just fetch the gague